### ISSUE TITLE
[TypeScript] Bug: Additional `_` being added to import of `__`

### ISSUE THREAD

**Issue by peterjuras** [renke/import-sort#18]
[From issue in vsc-sort-imports](https://github.com/peterjuras/vsc-sort-imports/issues/14)

## To Reproduce
Create a file `foo.ts` and add 
```typescript
import { __ } from 'ramda'
```

## Expected Results

Sorts imports as usual

## Actual Results

With each time I save an additional `_` is appended to the import

1 save 
```typescript
import { ___ } from 'ramda'
```
2 saves 
```typescript
import { ____ } from 'ramda'
```
and so on

**Comment by renke** [comment]
HI @peterjuras 

that's strange. I can reproduce that one, but I don't know how to fix it (without treating it as special case). 

If you go to https://astexplorer.net/ and enter your the code (while choosing TypeScript as parser), you will see that it recognizes `__` as `___`.

**Comment by renke** [comment]
The fix is now released with 3.0.
